cluster-scope/overlays/nerc-ocp-test-2/nodenetworkconfigurationpolicies/vlan-2175-nese.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: nmstate.io/v1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: vlan-2175-nese
+spec:
+  nodeSelector:
+    node-role.kubernetes.io/worker: ""
+  desiredState:
+    interfaces:
+    - name: bond0.2175
```

Comment:
I think you are missing the machineconfig resources that would create the `bond0` bond device.

Reply:
I know. I have asked about adding the machine configs to the templates before. So [this dir](https://github.com/OCP-on-NERC/nerc-ocp-config/tree/main/cluster-scope/overlays/nerc-ocp-prod/machineconfigs/configure-bond0) would need to be included in all overlays so can be added to the templates?

Reply:
The interface names differ from machine to machine (e.g., on the production cluster we have things arranged so that the interfaces are `nic1` and `nic2`, but on the obs cluster, they are `eno1` and `eno2`, etc, so the directory can't simply be copied verbatim. You would either need to template it, or add it manually.

Reply:
Okay, how do I figure out the interface names?

Reply:
Log into your work nodes and use the `ip` command to view interfaces. They should be identical for all your worker nodes (as long as all your workers are the same sort of machine).

Reply:
You can actually see the interfaces on the openshift console, they are all `eno1` and `eno2`